app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -266,6 +272,12 @@ class HomeFragment : Fragment() {
         return view
     }
 
+    override fun onConfigurationChanged(newConfig: Configuration) {
```

Comment:
nit: Should we also add a test similar to the one that we have on `BrowserFragmentTest`?

Reply:
Sure, not sure why I didn't see that.  I'll add the test.

Reply:
Sorry my bad I missed the test